### PR TITLE
Add JSDC event on this list

### PR DESCRIPTION
**Pull request by clonn**
* JSDC, JavaScript Developer Conference 2018 in Taiwan.
 * Nov 17.
 * Location: Taipei, Taiwan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ Conference | Dates | Proposals | Location | Twitter
 [DevFest Siberia](https://gdg-siberia.com) | Nov 23rd - 25th | [Jul 31st](https://cfp.exchange/cfp/details/devfest-siberia-2018) | Novosibirsk, Russia
 [NodeFest](https://nodefest.jp/2018/) | Nov 23th - 24th | [Sep 30th](https://docs.google.com/forms/d/1b3DFCCVIK1rinVMFO39q6hSUev1Q_27avmYRE2E4YG0/edit) | Tokyo, Japan | [![](https://user-images.githubusercontent.com/10602/44954380-6a3ab480-aea1-11e8-8ce7-ad540063f4de.png)](https://twitter.com/nodefest)
 [JSConf Colombia](https://jsconf.co) | Nov 16th - 17th | [Jul 31st](https://sessionize.com/jsconfco) | Medellin, Colombia | [![](https://user-images.githubusercontent.com/10602/44954380-6a3ab480-aea1-11e8-8ce7-ad540063f4de.png)](https://twitter.com/jsconfco)
+[JSDC-TW](https://js-poland.pl/) | Nov 17th | [Unknown](#) | Taipei, Taiwan | [![](https://user-images.githubusercontent.com/10602/44954380-6a3ab480-aea1-11e8-8ce7-ad540063f4de.png)](https://twitter.com/jsdc_tw)
```
Comment (CYBAI) [inline README.md]: I think the url should be http://2018.jsdc.tw/ ?

Comment (watson) [reply README.md]: Good point 😃 

Two other tiny nits: Can you move the line up above JSConf Colombia and just remove the link around "Unknown"? Then we're good 👌 

Thanks for contributing 💯

Comment (clonn) [reply README.md]: already updated, please check it, thanks
https://github.com/watson/conferences/pull/45/files

File: README.md [hunk 2]
```diff
@@ -45,6 +45,7 @@ Conference | Dates | Proposals | Location | Twitter
 [HolyJS](https://holyjs-moscow.ru/en/)* | Nov 24th - 25th | [Sep 24th](https://holyjs-moscow.ru/en/callforpapers/) | Moscow, Russia | [![](https://user-images.githubusercontent.com/10602/44954380-6a3ab480-aea1-11e8-8ce7-ad540063f4de.png)](https://twitter.com/HolyJSconf)
 [DevFest Siberia](https://gdg-siberia.com) | Nov 23rd - 25th | [Jul 31st](https://cfp.exchange/cfp/details/devfest-siberia-2018) | Novosibirsk, Russia
 [NodeFest](https://nodefest.jp/2018/) | Nov 23th - 24th | [Sep 30th](https://docs.google.com/forms/d/1b3DFCCVIK1rinVMFO39q6hSUev1Q_27avmYRE2E4YG0/edit) | Tokyo, Japan | [![](https://user-images.githubusercontent.com/10602/44954380-6a3ab480-aea1-11e8-8ce7-ad540063f4de.png)](https://twitter.com/nodefest)
+[JSDC-TW](http://2018.jsdc.tw/) | Nov 17th |  | Taipei, Taiwan | [![](https://user-images.githubusercontent.com/10602/44954380-6a3ab480-aea1-11e8-8ce7-ad540063f4de.png)](https://twitter.com/jsdc_tw)
```
Comment (watson) [inline README.md]: Thanks for updating the PR 

I'm sorry if I was unclear in my previous comment about the `unknown` link, what I ment was to replace `[unknown](#)` with just the string `unknown` in the 3rd column. I can also just do that my self to not bother you with it 😃

**Pull request merged**